lib/customer.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu'
+
+class Customer
+
+	attr_accessor :customer_order
+
+	def initialize
+		@customer_order = []
+	end
+
+
+ 	def place_order (choice)
+ 		menu = Menu.new
+ 		menu.view_menu.select do |key,value|
+			if key == choice
+				customer_order.push key => value
+			end
+		end
```

Comment:
`end` at 18, 2 is not aligned with `menu.view_menu.select do |key,value|` at 14, 3<br>Tab detected.